src/CosmosDB/CosmosDB.Test/ScenarioTests/RestoreTests.ps1

```diff
@@ -784,4 +784,54 @@ function Test-ProvisionCosmosDBAccountBackupPolicyWithContinuous7DaysCmdLets {
   Assert-NotNull $sourceRestorableAccount.DatabaseAccountName
   Assert-NotNull $sourceRestorableAccount.CreationTime
   Assert-NotNull $sourceRestorableAccount.OldestRestorableTime
+}
+
+function Test-CrossRegionRestoreAccountCmdlets {
+  #use an existing account with the following information
+  $rgName = "PSCosmosDBResourceGroup53"
```

Comment:
curious how the group number is picked? they look all over the place :)

Reply:
They are, I just picked a group number which was not there in this test file, otherwise I see they are all random ones.